newIDE/app/src/GameDashboard/GameDashboardCard.js

```diff
@@ -50,13 +51,37 @@ import PreferencesContext from '../MainFrame/Preferences/PreferencesContext';
 import { textEllipsisStyle } from '../UI/TextEllipsis';
 import FileWithLines from '../UI/CustomSvgIcons/FileWithLines';
 import TextButton from '../UI/TextButton';
-import { Tooltip } from '@material-ui/core';
+import { getRelativeOrAbsoluteDisplayDate } from '../Utils/DateDisplay';
 const electron = optionalRequire('electron');
 const path = optionalRequire('path');
 
 export const getThumbnailWidth = ({ isMobile }: {| isMobile: boolean |}) =>
   isMobile ? undefined : Math.min(245, Math.max(130, window.innerWidth / 4));
 
+export const getProjectDisplayDate = (i18n: I18nType, date: number) =>
+  getRelativeOrAbsoluteDisplayDate({
+    i18n,
+    dateAsNumber: date,
+    sameDayFormat: 'todayAndHour',
+    dayBeforeFormat: 'yesterdayAndHour',
+    relativeLimit: 'currentWeek',
+    sameWeekFormat: 'thisWeek',
+  });
+export const getDetailedProjectDisplayDate = (i18n: I18nType, date: number) =>
+  i18n.date(date, {
+    dateStyle: 'short',
+    timeStyle: 'short',
+  });
+
+const getNoProjectAlertMessage = () => {
+  if (!electron) {
+    // Trying to open a local project from the web app of the mobile app.
+    return t`Looks like your project isn't there!${'\n\n'}Your project is surely stored on your computer.`;
```

Comment:
I'd rather say `Your project must be stored on your computer` rather than surely, it feels more english